global.json

```diff
@@ -1,5 +1,5 @@
 {
     "sdk": {
-        "version": "5.0.301"
+        "version": "5.0.302"
```

Comment:
Also check/update: https://github.com/microsoft/fhir-server/blob/main/build/docker/Dockerfile and https://github.com/microsoft/fhir-server/blob/main/samples/templates/default-azuredeploy.json